Makefile

```diff
@@ -1,455 +1,126 @@
-# CrazyFlie's Makefile
-# Copyright (c) 2011-2021 Bitcraze AB
-# This Makefile compiles all the object file to ./bin/ and the resulting firmware
-# image in ./cfX.elf and ./cfX.bin
 
-CRAZYFLIE_BASE ?= ./
-
-# Put your personal build config in tools/make/config.mk and DO NOT COMMIT IT!
-# Make a copy of tools/make/config.mk.example to get you started
--include tools/make/config.mk
-
-CFLAGS += $(EXTRA_CFLAGS)
-
-######### JTAG and environment configuration ##########
 OPENOCD           ?= openocd
 OPENOCD_INTERFACE ?= interface/stlink.cfg
 OPENOCD_CMDS      ?=
-CROSS_COMPILE     ?= arm-none-eabi-
-PYTHON            ?= python3
-DFU_UTIL          ?= dfu-util
-CLOAD             ?= 1
-DEBUG             ?= 0
-CLOAD_CMDS        ?=
-CLOAD_ARGS        ?=
-PLATFORM          ?= cf2
-LPS_TDMA_ENABLE   ?= 0
-LPS_TDOA_ENABLE   ?= 0
-LPS_TDOA3_ENABLE  ?= 0
 
 # Cload is handled in a special way on windows in WSL to use the Windows python interpreter
 ifdef WSL_DISTRO_NAME
-CLOAD_SCRIPT      ?= python.exe -m cfloader
```

Comment:
So... now 'make cload' does not work on my WSL anymore because of this line change which has been reverted.

Reply:
Great  catch! Will re-add!

Reply:
or rather, it is still there, right? Just further up

```
# Cload is handled in a special way on windows in WSL to use the Windows python interpreter
ifdef WSL_DISTRO_NAME
PYTHON      ?= python.exe
else
PYTHON      ?= python3
endif

DFU_UTIL          ?= dfu-util

CLOAD_SCRIPT      ?= $(PYTHON) -m cfloader
```
Is this not the same?

Reply:
hmmm.. looking further in the makefile, it should have worked... but it still picking the python program from within WSL

Reply:
yeah it should be the same if I look at it visually.... but if I do make cload in wsl.... it pick just regular python3 and not python.exe. So there is something wrong here still

Reply:
perhaps @ataffanel can see what is going on here?

Reply:
Ah, it is because PYTHON is being set down in `Makefile.kbuild`. And since we go:

```
PYTHON ?= python.exe
```

It will have no effect if `$PYTHON` is set. I will fix so that we force it set when WSL detected.

Reply:
Fixed in latest commit